src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -130,11 +131,10 @@ public static double conditionalMutationsTime(ConditionalWriter cw, Long seq) th
     if (cmuts.size() != count) {
       throw new RuntimeException();
     }
-    long t2 = System.currentTimeMillis();
 
-    double rate = 10000 / ((t2 - t1) / 1000.0);
+    long t2 = System.nanoTime();
 
-    return rate;
+    return 10000 / TimeUnit.MICROSECONDS.toSeconds(t2 - t1);
```

Comment:
Should be `NANOSECONDS.toSeconds`, but otherwise, this looks much cleaner.

Reply:
I did nanoseconds to seconds first but it was returning 0 when the conversion was applied. Microseconds was the only one that returned nonzero number so I rolled with it. The results are comparable to the previous ones.

Reply:
NANOSECONDS is the correct units here. But, you'll have to cast either the numerator or denominator to a double to get a non-zero rate. Something like:

```java
return 10000.0 / TimeUnit.NANOSECONDS.toSeconds(t2-t1);
```
or
```java
return 10000 / (double) TimeUnit.NANOSECONDS.toSeconds(t2-t1);
```

Reply:
ah, that makes a lot of sense.I figured it was something to do with the return type but didn't know just adding the .0 at the end of the numerator would fix that. I will fix these tomorrow, thank you.